packages/githublight/README.md

```diff
@@ -0,0 +1,40 @@
+# Github Light
+
+A theme that mimics github light theme. It is heavily inspired in the `github light theme default` for VSCode available [here](https://github.com/primer/github-vscode-theme).
+
+## Usage ✨
+
+To enable the theme in JupyterLab: `Go to Settings -> JupyterLab Theme -> Github Light`
+
+## Colors
+
+All the colors used for text in the theme are compliant with [WCAG 2.1 standards](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
```

Comment:
```suggestion
All the colors used for text in the theme are compliant with [WCAG 2.1 contrast standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
```

Reply:
Is it AA/AAA - we should clarify here.
Also, what about non-text elements? Are these also WCAG-conformant?

Also let's use conformant over compliant as we are adhering to the standards rather than being certified or similar

Reply:
just saw the table below so perhaps indicating we have a table here would be useful

Reply:
> Also, what about non-text elements? Are these also WCAG-conformant?

These are harder to test and present here just because there's a lot of possible combinations around the UI. I can try to add them just to see how many there are.

Reply:
We do not need to add this to a table. 
But since we are saying that the themes are conformant we do need to make sure that this the case. Either through manual or automatic testing.


I think having the table was useful for the syntax highlighting themes as the colour combination are smaller but having it in this theme makes it slightly redundant. Will defer to @gabalafou for further discussion too